packages/fx-core/src/question/create.ts

```diff
@@ -1357,22 +1357,51 @@ export function officeAddinHostingQuestion(): SingleSelectQuestion {
   };
 }
 
-export function OfficeAddinFrameworkQuestion(): SingleSelectQuestion {
-  return {
+export function officeAddinFrameworkQuestion(): SingleSelectQuestion {
+  const officeAddinFrameworkQuestion: SingleSelectQuestion = {
     type: "singleSelect",
     name: QuestionNames.OfficeAddinFramework,
     cliShortName: "f",
     cliDescription: "Framework for WXP extension.",
     title: getLocalizedString("core.createProjectQuestion.projectType.officeAddin.framework.title"),
+    dynamicOptions: getAddinFrameworkOptions,
     staticOptions: [
       { id: "default", label: "Default" },
       { id: "react", label: "React" },
     ],
     placeholder: getLocalizedString(
       "core.createProjectQuestion.projectType.officeAddin.framework.placeholder"
     ),
-    default: "default",
+    default: (inputs: Inputs) => {
+      return getAddinFrameworkOptions(inputs)[0].id;
+    },
```

Comment:
Is this really necessary? I see this code will always return `"default"`.

Reply:
Yeah, it always return "default", just only not to write it by hard code.

Reply:
Sure, If this logic is keep for further use, please just resolve this comment.